src/ImageViewer/imageslist.tsx

```diff
@@ -0,0 +1,396 @@
+export interface Api {
+    id: number;
+    sol: number;
+    cameraName: string;
+    cameraFullName: string;
+    imageUrl: string;
+    earthDate: string;
+    roverName: string;
+}
```

Comment:
sorry - missed this before.
Could we give this a better name.

I don't like API much because:
a) its not an API - its an image that we got from the API
b) its going to be super confusing once we have more than 1 thing that we get from the API

Could you change it to something more descriptive?